src/circuit/gadget/ecc/chip.rs

```diff
@@ -221,9 +223,6 @@ impl EccChip {
         // mul_fixed::base_field_element::Config:
         // - [advices[6], advices[7], advices[8]]: canon_advices
         //
-        // mul::overflow::Config:
-        // - [advices[0], advices[1], advices[2]]: advices
-        //
```

Comment:
I believe this (now deleted) comment should have said `[advices[6], advices[7], advices[8]]`.